WalletWasabi.Gui/Global.cs

```diff
@@ -156,9 +156,10 @@ public async Task InitializeNoWalletAsync(TerminateService terminateService)
 						await TorManager.StartAsync(ensureRunning: true).ConfigureAwait(false);
 					}
 
-					var fallbackRequestTestUri = new Uri(Config.GetFallbackBackendUri(), "/api/software/versions");
-
-					HostedServices.Register(new TorMonitor(period: TimeSpan.FromSeconds(3), fallbackRequestTestUri, Config.TorSocks5EndPoint, TorManager), nameof(TorMonitor));
+					var httpClient = (WalletWasabi.Tor.Http.TorHttpClient)Synchronizer.HttpClientFactory.NewHttpClient(() => Config.GetFallbackBackendUri(), isolateStream: false);
+#pragma warning disable CA2000 // Dispose objects before losing scope
+					HostedServices.Register(new TorMonitor(period: TimeSpan.FromSeconds(3), httpClient, TorManager), nameof(TorMonitor));
```

Comment:
Do we need a TorMonitor if it is turned off?

Reply:
TorMonitor is registered here only when Tor is ON.